### PR TITLE
[DOCS] Indicator enhancements in 8.6

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2722 and some items in https://github.com/elastic/security-docs/issues/2721.

Previews:
- Indicators of compromise: Refreshed images in the topic and added three new sections:
  - [Attach indicators to cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#attach-indicator-to-case)
  - [Review indicator details in cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#review-indicator-in-case)
  - [Remove indicators from cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#delete-indicator-from-case)
- [Open and manage cases | Manage existing cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/cases-open-manage.html#cases-ui-manage): Condensed list and updated bullet # 3 to let users know they could examine indicators attached to a case.

### CODE REVIEW

File: docs/cases/indicators-of-compromise.asciidoc [hunk 1]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+[role="screenshot"]
+image::images/indicator-added-to-case.png[Shows an indicator attached to a case]
+
+[discrete]
+[[review-indicator-in-case]]
+=== Review indicator details in cases
+
+When you attach an indicator to a case, the indicator is added a new comment with the following details:
+
+* *Indicator name*: The name of the indicator. Clicking on the linked name opens the Indicator details flyout, which contains additional indicator metadata in JSON format.
```
Comment (nastasha-solomon) [inline docs/cases/indicators-of-compromise.asciidoc:118]:
```suggestion
* *Indicator name*: The name of the indicator. Clicking on the linked name opens the Indicator details flyout, which contains additional information about the indicator. Indicator details are provided in JSON format.
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 2]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+* *Feed name*: The name of the threat feed that the indicator was ingested from.
```
Comment (nastasha-solomon) [inline docs/cases/indicators-of-compromise.asciidoc:119]:
```suggestion
* *Feed name*: The name of the threat feed from which the indicator was ingested.
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 3]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+== Attach indicators to cases
+
+Provide more context to your investigations by attaching indicators to cases:
+
+. From the Indicators table and click the *More actions menu* *(…​)*. Alternatively, open an indicator's details, then select *Take action*.
+. Select one of the following:
+.. *Add to existing case*: From the *Select case* dialog box, select the case to which you want to attach the indicator. 
+.. *Add to new case*: Configure the case details. Refer to <<cases-ui-open>> to learn more about opening a new case. 
+
+The indicator is added to the case as a new comment.
```
Comment (joepeeples) [inline docs/cases/indicators-of-compromise.asciidoc:107]: A couple small Asciidoc fixes: the options should be bullets, not ordered steps, and the final line should be a continuation of step 2, not main body text. The bullets also have to be encased in an [open block](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#attach-to-ancestor-list) (`--`) to get the correct alignment. (I tested this syntax locally so it should work, but might want to double-check on your end too.)
```suggestion
. Select one of the following:
+
--
* *Add to existing case*: From the *Select case* dialog box, select the case to which you want to attach the indicator. 
* *Add to new case*: Configure the case details. Refer to <<cases-ui-open>> to learn more about opening a new case. 
--
+
The indicator is added to the case as a new comment. 
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 4]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+image::images/indicator-added-to-case.png[Shows an indicator attached to a case]
+
+[discrete]
+[[review-indicator-in-case]]
+=== Review indicator details in cases
+
+When you attach an indicator to a case, the indicator is added a new comment with the following details:
+
+* *Indicator name*: The name of the indicator. Clicking on the linked name opens the Indicator details flyout, which contains additional information about the indicator. Indicator details are provided in JSON format.
+* *Feed name*: The name of the threat feed from which the indicator was ingested.
```
Comment (joepeeples) [inline docs/cases/indicators-of-compromise.asciidoc:119]: Maybe omit some of the more repetitive details, make the actions more straightforward:
```suggestion
* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains additional information about the indicator. Indicator details are in JSON format.
* *Feed name*: The threat feed from which the indicator was ingested.
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 5]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+* *Indicator type*: The indicator type, for example, `file` or `.exe`.
+
+[discrete]
+[[delete-indicator-from-case]]
+=== Remove indicators from cases
+To remove an indicator attached to a case, click the *More actions* menu *(…​)* in the case comment, then click *Delete attachment*.
```
Comment (joepeeples) [inline docs/cases/indicators-of-compromise.asciidoc:125]: A little less wordy:
```suggestion
To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 6]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+
+. From the Indicators table and click the *More actions menu* *(…​)*. Alternatively, open an indicator's details, then select *Take action*.
+. Select one of the following:
+.. *Add to existing case*: From the *Select case* dialog box, select the case to which you want to attach the indicator. 
+.. *Add to new case*: Configure the case details. Refer to <<cases-ui-open>> to learn more about opening a new case. 
+
+The indicator is added to the case as a new comment. 
+
+[role="screenshot"]
+image::images/indicator-added-to-case.png[Shows an indicator attached to a case]
```
Comment (joepeeples) [inline docs/cases/indicators-of-compromise.asciidoc:110]:
```suggestion
image::images/indicator-added-to-case.png[An indicator attached to a case]
```

**Review by joepeeples — COMMENTED** [review]
New content looks good! Added a few suggested line edits.

File: docs/cases/indicators-of-compromise.asciidoc [hunk 7]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
...
+
+[discrete]
+[[review-indicator-in-case]]
+=== Review indicator details in cases
+
+When you attach an indicator to a case, the indicator is added a new comment with the following details:
```
Comment (benironside) [inline docs/cases/indicators-of-compromise.asciidoc:116]:
```suggestion
When you attach an indicator to a case, the indicator is added as a new comment with the following details:
```

**Review by benironside — COMMENTED** [review]
LGTM, left one minor suggestion

File: docs/cases/indicators-of-compromise.asciidoc [hunk 8]
```diff
@@ -52,7 +52,7 @@ If indicator data is not appearing in the Indicators table after you installed a
 [[intelligence-page-ui]]
 == Indicators page UI
 
-After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the occurrence of indicators over time.
+After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the shows accumulative values in the legend.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cases/indicators-of-compromise.asciidoc:55]: Very close, this should actually be "total accumulative values".

```suggestion
After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the total accumulative values in the legend.
```

File: docs/cases/cases-manage.asciidoc [hunk 9]
```diff
@@ -46,19 +46,14 @@ image::images/cases-home-page.png[Case UI Home]
...
+* <<cases-manage-comments, Add and manage comments>>
 +
 TIP: Comments can contain Markdown. For syntax help, click the Markdown icon (image:images/markdown-icon.png[Click markdown icon,17,17]) in the bottom right of the comment.
 
-* <<cases-examine-alerts,Examine alerts attached to the case>>
+* Examine <<cases-examine-alerts,alerts>> and <<review-indicator-in-case,indicators>> attached to the case
 * <<cases-lens-visualization>>
-* Modify the case's description
-* Add or remove assignees
+* Modify the case's description, assignees, severity, status, and more
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cases/cases-manage.asciidoc:55]: Good point, the list is already pretty long. 

```suggestion
* Modify the case's description, assignees, severity, status, and tags. 
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 10]
```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
 `file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.
 
 Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field-value pairs.
+
+[discrete]
+[[attach-indicator-to-case]]
+== Attach indicators to cases
+
+Provide more context to your investigations by attaching indicators to cases:
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cases/indicators-of-compromise.asciidoc:100]: @dhru42 sure, I can work this in. I'm always a fan of describing the value a new feature provides so I have no objections to working this in. How about something like:

Attaching indicators to cases provides more context and actionability to your investigations. This also allows you to share or escalate threat intelligence to other teams or workflows.

[the remark this answers is not in the document]
Comment (dhru42) [reply docs/cases/indicators-of-compromise.asciidoc:100]: Perfect, thank you @nastasha-solomon

[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cases/indicators-of-compromise.asciidoc:100]:
```suggestion
Attaching indicators to cases provides more context and actionability to your investigations. In doing this, you can easily share or escalate threat intelligence to other teams or workflows.

To add indicators to cases:
```

File: docs/cases/indicators-of-compromise.asciidoc [hunk 11]
```diff
@@ -52,7 +52,7 @@ If indicator data is not appearing in the Indicators table after you installed a
 [[intelligence-page-ui]]
 == Indicators page UI
 
-After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the occurrence of indicators over time.
+After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the total accumulative values in the legend.
```
Comment (nastasha-solomon) [inline docs/cases/indicators-of-compromise.asciidoc:55]:
```suggestion
After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the total values in the legend.
```

**Comment by PhilippeOberti** [comment]
@nastasha-solomon the text looks good, only thing that I want to clarify is related to the flyout: in 8.6 we will NOT have the full flyout with the 3 tabs (Overview, Table and Json). We will have a simpler flyout, with only a json view of the indicator.

We didn't have time to merge the code for the full-blown flyout before the feature freeze, so it will go live in 8.7.

**Comment by nastasha-solomon** [comment]
Thanks for the additional information, @PhilippeOberti ! I'll file a doc issue to update the description for the Indicator name field ([here](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#review-indicator-in-case)) in 8.7.

**Comment by karanbirsingh-qasource** [comment]
Hi @nastasha-solomon 

we have validated the Doc Links under the preview and all the details are correct and good to go .

**screen-shot**

- Indicators of compromise: Refreshed images in the topic and added three new sections:
  - [Attach indicators to cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#attach-indicator-to-case)

  - [Review indicator details in cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#review-indicator-in-case)

  - [Remove indicators from cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html#delete-indicator-from-case)

image

- [Open and manage cases | Manage existing cases](https://security-docs_2766.docs-preview.app.elstc.co/guide/en/security/master/cases-open-manage.html#cases-ui-manage): Condensed list and updated bullet # 3 to let users know they could examine indicators attached to a case. 


image


Hence we marking this PR as QA Validated.

Thanks!

**Review by PhilippeOberti — APPROVED** [review]

File: docs/cases/indicators-of-compromise.asciidoc [hunk 12]
```diff
@@ -92,3 +92,42 @@ A new Timeline opens with an automatically populated KQL query. The query contai
 `file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.
 
 Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field-value pairs.
+
+[discrete]
+[[attach-indicator-to-case]]
+== Attach indicators to cases
+
+Attaching indicators to cases provides more context and actionability to your investigations. In doing this, you can easily share or escalate threat intelligence to other teams or workflows.
```
Comment (jmikell821) [inline docs/cases/indicators-of-compromise.asciidoc:100]: Simplifying and tightening up just a bit here.
```suggestion
Attaching indicators to cases provides more context and available actions for your investigations. This feature allows you to easily share or escalate threat intelligence to other teams.
```

**Review by jmikell821 — COMMENTED** [review]
Previous feedback IMO was good. I just have one small comment, thanks!

**Pull request merged**